pkg/client/cache/listers.go

```diff
@@ -58,17 +59,92 @@ func (s *StoreToWorkflowLister) GetJobWorkflows(job *k8sBatch.Job) (workflows []
 	}
```

Comment:
May be weird this function is plural although we only expect one or zero workflows to be returned